grafana/dashboards/internal/charge-details.json

```diff
@@ -234,7 +234,7 @@
           "group": [],
           "metricColumn": "none",
           "rawQuery": true,
-          "rawSql": "SELECT charge_energy_added from charging_processes where id = $charging_process_id;",
+          "rawSql": "SELECT charge_energy_added from charges where id = $charging_process_id order by date desc fetch first 1 row only ;",
```

Comment:
`id` should be `charging_process_id`

Reply:
In rare cases `charge_energy_added` does not start a zero so that the total energy added is less than the last reported `charge_energy_added` (`end_charge_energy_added - start_charge_energy_added` to be precise). Therefore we should still prefer to display the value reported in the `charging_processes` table, for example:

```SQL
WITH completed_charge AS (
	SELECT charge_energy_added
	FROM charging_processes
	WHERE id = $charging_process_id
),
in_progress_charge AS (
	SELECT charge_energy_added
	FROM charges
	WHERE charging_process_id = $charging_process_id
	ORDER BY date DESC FETCH FIRST 1 ROW ONLY
)
SELECT COALESCE(completed_charge.charge_energy_added, in_progress_charge.charge_energy_added)
FROM completed_charge
JOIN in_progress_charge on true
```

Reply:
Thanks, I messed up my repor and had closed out this branch... so I couldnt fix it. I will close this PR as I made a new one.

https://github.com/adriankumpf/teslamate/pull/744